backend/requirements.txt

```diff
@@ -12,3 +12,4 @@ neo4j-backup==0.2.4
 pytest>=7.2.0
 requests>=2.28.1
 python-decouple>=3.6
```

Comment:
python-decouple is mentioned twice!

Reply:
Thanks for the review @mohanqxf2 . I have fixed the requirements file. 
For the second comment, i don't see a tests folder under the new branch https://github.com/qxf2/qxf2-survey/tree/Add_pre_run_script_for_tests. I guess the issue is git doesn't recognize empty directories under version control. You could try re-cloning the repo.